libethcore/Miner.h

```diff
@@ -79,36 +79,17 @@ struct HwMonitor
     double powerW = 0;
 };
 
-inline std::ostream& operator<<(std::ostream& os, HwMonitor _hw)
-{
-    os << _hw.tempC << "C " << _hw.fanP << "%";
-    if (_hw.powerW)
-        os << ' ' << fixed << setprecision(0) << _hw.powerW << "W";
-    return os;
-}
+std::ostream& operator<<(std::ostream& os, HwMonitor _hw);
 
-class FormatMemSize
+class FormattedMemSize
 {
 public:
-    FormatMemSize() = delete;
-    FormatMemSize(uint64_t s) { m_size = s; }
+    FormattedMemSize() = delete;
```

Comment:
This is not needed if you declared any other constructor.

Reply:
No default constructor if any other constructor is declared? I did not know that!